### PR TITLE
Feature - New pricing collection

### PR DESCRIPTION
**Pull request by usman-faisal**

### CODE REVIEW

File: public/components/marketing-pricings/1.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                <div class="flex items-center gap-2 justify-center">
+                    <strong class="text-4xl">20$</strong>
+                    <span> / month</span>
+                </div>
+            </div>
+            <ul class="text-left flex flex-col gap-2">
+                <li class="flex gap-1 items-center">
+                    <svg width="18" height="18" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M42.875 8.625C42.8438 8.63281 42.8125 8.64453 42.7812 8.65625C42.5195 8.72266 42.293 8.89063 42.1562 9.125L21.7188 40.8125L7.65625 28.125C7.41016 27.8125 7 27.6758 6.61328 27.7773C6.22656 27.8789 5.94141 28.2031 5.88281 28.5977C5.82422 28.9922 6.00391 29.3828 6.34375 29.5938L21.25 43.0938C21.4688 43.2852 21.7617 43.3711 22.0508 43.3281C22.3398 43.2852 22.5938 43.1211 22.75 42.875L43.8438 10.1875C44.0742 9.85938 44.0859 9.42578 43.875 9.08594C43.6641 8.74609 43.2695 8.56641 42.875 8.625Z" fill="#4F46FF"/>
+                    </svg>
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:15]: Should be using Heroicons for any icons - https://heroicons.com

File: public/components/marketing-pricings/1.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<section>
+    <div class="flex justify-center items-center gap-6 flex-wrap py-16 sm:px-6 lg:px-8">
+        <div class="text-center shadow-sm border border-gray-200 flex flex-col gap-6 sm:p-10 lg:p-12 p-8 rounded-2xl">
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:3]: Padding is too much on mobile

File: public/components/marketing-pricings/1.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            <div class="flex flex-col gap-3">
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:4]: Doesn't need to be using flex

File: public/components/marketing-pricings/1.html (hunk 2 above)
Comment (markmead) [reply public/components/marketing-pricings/1.html:3]: Doesn't need to be using flex

File: public/components/marketing-pricings/1.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                <p class="text-3xl">Starter</p>
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:5]: Font is too big on mobile

File: public/components/marketing-pricings/1.html [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                <div class="flex items-center gap-2 justify-center">
+                    <strong class="text-4xl">20$</strong>
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:7]: Font is too big on mobile

File: public/components/marketing-pricings/1.html [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+                    <span> / month</span>
+                </div>
+            </div>
+            <ul class="text-left flex flex-col gap-2">
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:11]: Doesn't need to be using flex

File: public/components/marketing-pricings/1.html (hunk 2 above)
Comment (markmead) [reply public/components/marketing-pricings/1.html:3]: Shouldn't set the text position here, do it at an element group level

File: public/components/marketing-pricings/1.html [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+                <li class="flex gap-1 items-center">
+                    <svg width="18" height="18" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M42.875 8.625C42.8438 8.63281 42.8125 8.64453 42.7812 8.65625C42.5195 8.72266 42.293 8.89063 42.1562 9.125L21.7188 40.8125L7.65625 28.125C7.41016 27.8125 7 27.6758 6.61328 27.7773C6.22656 27.8789 5.94141 28.2031 5.88281 28.5977C5.82422 28.9922 6.00391 29.3828 6.34375 29.5938L21.25 43.0938C21.4688 43.2852 21.7617 43.3711 22.0508 43.3281C22.3398 43.2852 22.5938 43.1211 22.75 42.875L43.8438 10.1875C44.0742 9.85938 44.0859 9.42578 43.875 9.08594C43.6641 8.74609 43.2695 8.56641 42.875 8.625Z" fill="#4F46FF"/>
+                    </svg>
+                    <span>
+                        Help center access
+                    </span>
+                </li>
+            </ul>
+            <a href="#" class="bg-indigo-600 text-white py-4 px-3 rounded self-center">Get Started</a>
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:45]: Use an existing button style from HyperUI - https://www.hyperui.dev/components/marketing/buttons

**Review by markmead — CHANGES_REQUESTED** [review]
Only reviewed one section as the feedback can be applied to more. 

I don't think we need two components that are basically the same, best to decide on one.

**Comment by usman-faisal** [comment]
I've made the necessary changes, I am using flex in different place to avoid using margin or for spacing and aligning items. Let me know what I can fix/improve next! Thank you so much for your help ♥

**Comment by usman-faisal** [comment]
> Only reviewed one section as the feedback can be applied to more. 
> 
> 
> 
> I don't think we need two components that are basically the same, best to decide on one.
> 
> 
> 
> ---
> 
> 
> 
> Good work so far, looks nice!

I've made the necessary changes, I am using flex in different place to avoid using margin or for spacing and aligning items. Let me know what I can fix/improve next! Thank you so much for your help ♥

**Comment by markmead** [comment]
Not all feedback has been addressed as I'd rather see classes like `mt` and `space-y` used instead of `flex` for vertical spacing. It's unnecessary to be using `flex flex-col gap-3` when you can just do `space-y-3`, as an example.

File: public/components/marketing-pricings/1.html [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+        <div class="flex flex-col sm:gap-3 gap-1">
+            <p class="text-2xl sm:text-3xl">PRO</p>
+            <div class="flex items-center gap-2 justify-center">
+                <strong class="text-3xl sm:text-4xl">30$</strong>
+                <span> / month</span>
+            </div>
+        </div>
+        <ul class="flex flex-col gap-2 sm:text-base text-sm">
+            <li class="flex gap-1 items-center">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="#ffffff" class="w-5 h-5">
```
Comment (markmead) [inline public/components/marketing-pricings/1.html:61]: Thanks for changing to Heroicons, but `stroke` should be using `currentColor` not a hardcoded value.

**Review by markmead — CHANGES_REQUESTED** [review]

**Comment by usman-faisal** [comment]
Hey! I've made the changes. Also thanks for helping me out on eliminating `flex flex-col gap-3`. Let me know if there are any other possible improvements that can be made ❤

**Comment by markmead** [comment]
Thanks @usman-faisal, I'll pull down the PR and have a look!

**Comment by markmead** [comment]
Thanks for working on this @usman-faisal

**Pull request merged**